cypress/support/step_definition/course/createAndDeleteCourseSteps.spec.js

```diff
@@ -0,0 +1,46 @@
+import Courses from '../../pages/course/pageCourses'
+
+const courses = new Courses()
+
+//Scenario: Adding a new course
```

Comment:
here the comments are in between because it has the same flow as in the feature file, so to read in a same sequence the user user journey as in the feature file, if we put all comments above and then the user flow would be changed as well, so we can adapt that in a follow up then once we have all features & step definition merged based on this structure.

Reply:
created follow-up ticket on adapting second version of step-definition file (#2035).